reference/conanfile/attributes/binary_model.inc

```diff
@@ -331,6 +331,33 @@ For example:
             "option2": "Describe the purpose and functionality of 'option2'. ",
         }
 
+languages
+---------
+
+.. include:: ../../common/experimental_warning.inc
+
+
+From Conan 2.4, the ``conanfile.py`` recipe attribute ``languages`` can be used to define the programming languages 
+involved in this package. At the moment the ``C`` and ``C++`` languages are the possible values. For example a
+pure C package would define something as:
+
+.. code-block:: python
+
+    class ZLib(ConanFile):
+        languages = "C"
+
+
+It is possible to define more than one language, for example ``languages = "C", "C++"`` is the correct definition when
+a package is built from both C and C++ sources.
+
+Regarding ``languages`` definition, the following will happen:
+
+- If no ``languages`` is defined or ``C`` is not a declared language, ``compiler.cstd`` subsetting will be automatically removed 
+  at package ``configure()`` time (to achieve backward compatibility).
```

Comment:
```suggestion
  at package ``configure()`` time (to achieve backward compatibility).
  Thus Conan assumes that ``C++`` is the default value for ``languages`` if not overriden by the user.
```

Reply:
Not super accurate, it will not be reported ``languages = "C++"`` if it is not defined in any of the places it is reported (json output, ``dependencies`` interface, etc), it will be an empty list. So it is not the default value, the default value is an empty list.